corehq/apps/users/management/commands/clone_web_users.py

```diff
@@ -75,6 +75,12 @@ def create_new_user_from_old_user(old_user, new_username):
     return WebUser.get_by_username(new_user.username, strict=True)
 
 
+def deactivate_user(user):
+    user.is_active = False
```

Comment:
Thanks for raising this. It looks like we include the ModelBackend in AUTHENTICATION_BACKENDS which the docs mention enforce the `is_active` flag. Also while locally testing, I confirmed that my "old user" was logged out after running this command because of this flag. It is interesting to think that if in the future we swap out django authentication backends for whatever reason, inactive users would have to be dealt with properly. It seems safe enough to assume we will handle that situation appropriately if/when we get there.